CPP/Algorithm/N-QUEEN.cpp

```diff
@@ -0,0 +1,71 @@
+Input: The chess board, the col where the queen is trying to be placed.
+
```

Comment:
Please follow the existing code standard. You didn't comment out the statements which is not the part of C++. 
It looks like you have copy pasted it from tutorials points because I have found exact similar code here [N-Queen Problem](https://www.tutorialspoint.com/N-Queen-Problem).

Reply:
Let me review it once and i will fix it .